### ISSUE TITLE
[Revamp Landing Screen] JPAndroid - Font size of looping text should be fixed

### ISSUE THREAD

**Issue by ovitrif** [wordpress-mobile/WordPress-Android#17299]
Font size of the animated large text should be fixed and not increase/decrease based on font size Accessibility settings.

**Repro**

1. Make sure `LandingScreenRevamp` feature flag is enabled

<details>
<summary>How to Enable the <code>LandingScreenRevamp</code> feature flag</summary>

- Either `login` → `Me` → `App Settings` → `Debug Settings` → enable `LandingScreenRevamp` → `RESTART THE APP` → `logout` or
- Apply this hard-coded solution:

<details>
<summary>Patch to enable the <code>LANDING_SCREEN_REVAMP</code> build flag</summary>

```diff
diff --git a/WordPress/build.gradle b/WordPress/build.gradle
index 63260a4ca8..fe999fa92f 100644
--- a/WordPress/build.gradle
+++ b/WordPress/build.gradle
@@ -112,7 +112,7 @@ android {
         buildConfigField "boolean", "READER_COMMENTS_MODERATION", "false"
         buildConfigField "boolean", "SITE_INTENT_QUESTION", "true"
         buildConfigField "boolean", "SITE_NAME", "false"
-        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "false"
+        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "true"
         buildConfigField "boolean", "LAND_ON_THE_EDITOR", "false"
         buildConfigField "boolean", "BLOGGING_PROMPTS", "false"
         buildConfigField "boolean", "QUICK_START_EXISTING_USERS_V2", "false"
```  
</details>

---

</details>

2. Launch Jetpack App and make sure you're logged out
3. Set font size to max large / small via Accessibility settings on your device
4. **Notice** The font size of the animated text is also increased / decreased
